src/components/Header/navbar.ts

```diff
@@ -16,8 +16,8 @@ const navbarItems = [
     },
     {
       label: "Blog",
-      // href: "/blog",
-      href: EXTERNAL_LINKS.OPENEBS_BLOGS,
+      href: "/blog",
+      // href: EXTERNAL_LINKS.OPENEBS_BLOGS,
```

Comment:
Let's remove commented code here as we no more need the external source anymore.

Reply:
remove unused code.